cmd/bosun/expr/expr.go

```diff
@@ -460,6 +454,17 @@ func (e *State) walkBinary(node *parse.BinaryNode, T miniprofiler.Timer) *Result
 }
 
 func operate(op string, a, b float64) (r float64) {
+	// Test short circuit before NaN.
+	switch op {
+	case "||":
+		if a != 0 {
```

Comment:
I think it is, because we want `1 || NaN` to also resolve to 1, and `||` should be symmetric.